src/v2/ion/i18nTransformer.js

```diff
@@ -128,6 +128,8 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.session.expired' : 'oie.idx.session.expired',
 
   'activate-device.userCode' : 'oie.device.code.activate.label',
+
+  'tooManyRequests' : 'oie.tooManyRequests',
```

Comment:
I don't think you need this change. This is only for input labels that are sent from the backend.